### ISSUE TITLE
pypi package doesn't include python files

### ISSUE THREAD

**Issue by matt-carr** [argoproj-labs/argo-client-python#21]
```~/workspace
❯ pip install --no-cache argo-workflows
Collecting argo-workflows
  Downloading argo_workflows-4.0.0-py3-none-any.whl (8.9 kB)
<<snip>>
Installing collected packages: argo-workflows
Successfully installed argo-workflows-4.0.0


~/workspace
❯ pip show -f argo-workflows
Name: argo-workflows
Version: 4.0.0
Summary: Community Maintained Python client for Argo Workflows
Home-page: https://github.com/argoproj-labs/argo-client-python
Author: Yudi Xue, Marek Cermak
Author-email: binarycrayon@gmail.com, macermak@redhat.com
License: MIT
Location: <<snip>>
Requires: kubernetes
Required-by:
Files:
  argo_workflows-4.0.0.dist-info/INSTALLER
  argo_workflows-4.0.0.dist-info/LICENSE
  argo_workflows-4.0.0.dist-info/METADATA
  argo_workflows-4.0.0.dist-info/RECORD
  argo_workflows-4.0.0.dist-info/REQUESTED
  argo_workflows-4.0.0.dist-info/WHEEL
  argo_workflows-4.0.0.dist-info/top_level.txt

~/workspace
❯
```

Works just fine with 3.6.1

**Comment by yxue-kabam** [comment]
Thanks for bringing this up -- I've released argo-workflows 4.0.1 which should now include all the files , could you try it again?

Sorry for the trouble

**Comment by matt-carr** [comment]
All good! Appreciate the work :)
